tfhe/benches/core_crypto/pbs_bench.rs

```diff
@@ -66,17 +66,49 @@ fn multi_bit_benchmark_parameters<Scalar: Numeric>(
 ) -> Vec<(String, (CryptoParametersRecord, LweBskGroupingFactor))> {
     if Scalar::BITS == 64 {
         vec![
+            (
+                "2_bits_multi_bit_group_2".to_string(),
+                (
+                    CryptoParametersRecord {
+                        lwe_dimension: Some(LweDimension(764)),
+                        lwe_modular_std_dev: Some(StandardDev(0.000006025673585415336)),
+                        pbs_base_log: Some(DecompositionBaseLog(18)),
+                        pbs_level: Some(DecompositionLevelCount(1)),
+                        glwe_dimension: Some(GlweDimension(3)),
+                        glwe_modular_std_dev: Some(StandardDev(0.0000000000039666089171633006)),
+                        polynomial_size: Some(PolynomialSize(1 << 9)),
```

Comment:
I'm curious about why you're shifting bits instead of putting the numerical value?

Reply:
parameters are given as the log of the polynomial size, it's simpler to shift by the log(N) as 1 << log(N) == N for N a power of 2, rather than compute it in my head and make a mistake 🙂